packages/calcite-components/src/components/accordion/accordion.tsx

```diff
@@ -116,6 +145,9 @@ export class Accordion {
   //  Private State/Props
   //
   //--------------------------------------------------------------------------
+  mutationObserver = createObserver("mutation", () => this.passPropsToAccordionItems());
+
+  @State() accordionItems: HTMLCalciteAccordionItemElement[] = [];
```

Comment:
I don't think this needs to be state since its not rendering.

Reply:
there's already a var that can be used for the items `this.items` instead of `accordionItems`